Physics/Dynamics/FormFactor.hpp

```diff
@@ -120,6 +122,12 @@ inline double FormFactor(std::complex<double> qValue, unsigned int orbitL,
     // Reference: S.U.Chung Annalen der Physik 4(1995) 404-430
     // z = q / (interaction range). For the interaction range we assume
     // 1/mesonRadius
+
+    // TODO: is this comment actually accurate?
```

Comment:
I think you put that here

Reply:
Ups, don't know how that ended up there... This comment comes from FormFactorDecorator.hpp